File: Api/SecurityDoctor/SecurityDoctor/DataBase/Notification.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;
using System.Xml.Serialization;

namespace SecurityDoctor.DataBase
{
    [Table("T_Notification")]
    public class Notification
    {
        public int id { get; set; }
        public string message { get; set; }
        public int userid { get; set; }
        public int status { get; set; }
        public int schedule { get; set; }
    }
}

File: Mobile/lib/Modules/notification.dart
class NotificationModel {
  int? id;
  String? message;
  int? userid;
  int? status;
  int? schedule;

  NotificationModel({this.id, this.message, this.userid, this.status,this.schedule});

  NotificationModel.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    message = json['message'];
    userid = json['userid'];
    status = json['status'];
    schedule = json['schedule'];
  }
}
